components/popover.scss

```diff
@@ -0,0 +1,8 @@
+$fd-icons-path: "../scss/icons/";
+$fd-fonts-path: "../scss/fonts/";
+
+@import "../scss/components/popover";
+@import "../scss/components/menu";
+@import "../scss/components/image";
```

Comment:
This is another component that is a little strange to apply self-contained styles to because the majority of the style content loaded here is for components other than the popover.  I know these were added because of the playground examples, but honestly the playground examples should be reworked to not include any specific fundamental-related content.  Images and menus shouldn't be used as those are completely self-contained themselves. The playground examples can show a plain element to anchor the popover to. Similarly, the menu content in the popover should be taken out.  It can use a plain `<ul>` with `<li>` elements, but should not have any fd-menu classes.

On the main docs site, we can keep the images and menus in place as those are more use case-related.